### PR TITLE
Feature/shield update2.0

### PR DESCRIPTION
**Pull request by vidalmelero**
Whats new?

New shield bar style

### CODE REVIEW

File: Scripts/src/Script/LightShield.cpp [hunk 1]
```diff
@@ -160,7 +171,7 @@ void LightShield::CastShield()
 void LightShield::UncastShield()
 {
 	active = false;
-
+	/*ui_manager->ShowShieldBar(active || broken, static_cast<uint32_t>(player_controller->player));*/
```
Comment (Dartemiss) [inline Scripts/src/Script/LightShield.cpp:174]: if not used delete it

**Review by Dartemiss — APPROVED** [review]
GJ! I would say that I don't like that the bar is on your face when shield is destroyed recovering. I would prefer an icon on the hud that represents that without disturbing the gameplay. But it is just my opinion so I aprove!

**Comment by luismoyano** [comment]
Aside from Enrique's comments, it looks good to me

**Review by luismoyano — APPROVED** [review]
LGTM

**Pull request merged**